config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -40,7 +40,7 @@ presubmits:
       preset-bazel-remote-cache-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190328-ccf146b96-experimental
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-experimental
```

Comment:
Should we remove the experimental part here?

Reply:
For any particular reason? We've run test-infra on `experimental` for as long as I recall (which is at least several months).